s3gw/README.md

```diff
@@ -0,0 +1,59 @@
+s3gw
+====
+
+s3gw is a minimal gateway implementation for Ceph RGW API.
+
+Environment variables
+---------------------
+
+- `BUCKET_HOST`: object bucket API endpoint host
+- `BUCKET_PORT`: object bucket API endpoint port
+- `BUCKET_NAME`: object bucket name
+- `BUCKET_REGION`: object bucket region
+- `AWS_ACCESS_KEY_ID`: access key ID
+- `AWS_SECRET_ACCESS_KEY`: access key secret
+
+`BUCKET_HOST`,  `BUCKET_NAME`, `AWS_ACCESS_KEY_ID` and `AWS_SECRET_ACCESS_KEY` are required. `BUCKET_PORT` and `BUCKET_REGION` are optional.
+
+Options
+-------
+
+- `--listen`: addr:port to listen to (passed to http.Server.Addr)
+- `--use-path-style`: use path-style bucket name
+- `--hosts-allow`: subnets allowed to access to this gw, separated by comma
+- `--hosts-deny`: subnets denied to access to this gw, separated by comma
+  - note: `--hosts-allow` and `--hosts-deny` control access to `/bucket/` only. `/health` and `/metrics` accept access from everywhere.
+
+Supported API
```

Comment:
Could you elaborate on the API specification such as response format, parameters if any, and support methods for `/health`, `/metrics`, and `/bucket` (Get only perhaps)? Also, It would be helpful to describe what metrics will be exported.